examples/timer_escalation/README.md

```
@@ -0,0 +1,104 @@
+A real deployment calls `tick` from cron or a `time.Ticker` with `time.Now()`. This
+example hands it a fixed clock advanced day by day so the output is reproducible.
+
+## Running the example
+
+```bash
+go run ./examples/timer_escalation
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)